beets/dbcore/query.py

```diff
@@ -447,6 +447,34 @@ def match(self, item):
         return any([q.match(item) for q in self.subqueries])
 
 
+class NotQuery(Query):
+    """A query that matches the negation of its `subquery`, as a shorcut for
+    performing `not(subquery)` without using regular expressions."""
+    def __init__(self, subquery):
+        self.subquery = subquery
+
+    def clause(self):
+        clause, subvals = self.subquery.clause()
+        if clause:
+            return 'not ({0})'.format(clause), subvals
+        else:
+            # special case for RegexpQuery, (None, ())
```

Comment:
That's a good point; I forgot that reading the `fast` flag would assume a FieldQuery.

Let's keep it like this. I suggest a comment something like this:

> If there is no clause, there is nothing to negate. All the logic is handled by `match()` for slow queries.